client/js/components/home/assessmentCreationForm.js

```diff
@@ -6,12 +6,21 @@ var TemplateSelect = require('./templateSelect')
 var $ = require('jquery')
 
 var AssessmentCreationForm = React.createClass({
+  propTypes: {
+    initialTags: React.PropTypes.array,
+    team: React.PropTypes.object
+  },
   getInitialState: function () {
     return {
       template: '',
       tags: ''
     }
   },
+  componentDidMount: function () {
+    this.setState({
+      tags: this.props.initialTags
+    })
+  },
```

Comment:
Thanks. Turns out I needed componentWillMount, and I needed to reformat the values I was passing.